apps/admin/frontend/src/screens/tally/import_election_results_reporting_file_modal.tsx

```diff
@@ -133,21 +127,7 @@ export function ImportElectionsResultReportingFileModal({
         title="No USB Drive Detected"
         content={<P>Insert a USB drive in order to import a results file.</P>}
         onOverlayClick={onClose}
-        actions={
-          <React.Fragment>
-            {window.kiosk && (
-              <FileInputButton
-                data-testid="manual-input"
-                onChange={processElectionResultReportingFileFromFilePicker}
-                accept=".json"
-                disabled
-              >
-                Select File…
-              </FileInputButton>
-            )}
```

Comment:
this button is just permanently disabled, so it can't matter too much...

Reply:
Oh lol good find